client/auth/handle-emailed-link-form.jsx

```diff
@@ -0,0 +1,172 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import emailValidator from 'email-validator';
+import request from 'superagent';
```

Comment:
At some point I'd aim to decouple the request creation (an action) from the view.